ls1intum/Artemis
# `General`: Add links to the Artemis documentation for more context

### Checklist
#### General
#### Client

### Motivation and Context


During a user study regarding usability of Artemis, it became apparent that especially new users don't understand the meaning of certain terms and what/how they can interact with them.
Since there is a documentation for Artemis, which provides a lot more context about the different parts of Artemis, which could help those users understand the platform better. Right now, accessing the documentation is not really obvious when using Artemis.

### Description

In order to make accessing the documentation easier, I added a help/information button to relevant elements (such as courses, exercises, exams, lectures, ...), which open the corresponding page in the documentation when being clicked.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to different pages such as courses, lectures, units, learning goals, exercises
3. Make sure the information button is visible 
4. Click the button
5. Make sure the corresponding page of the documentation opens

### Review Progress






#### Code Review
#### Manual Tests

### Screenshots


The information button

<img width="1739" alt="Bildschirmfoto 2022-11-02 um 09 49 34" src="https://user-images.githubusercontent.com/16115423/199515298-235cf045-104f-4afa-8a10-619e9eab5c93.png">

The opened documentation
<img width="1739" alt="Bildschirmfoto 2022-11-02 um 09 49 41" src="https://user-images.githubusercontent.com/16115423/199515344-1e027b56-9101-4f09-9676-a86c1a99ca03.png">
